docs/.vitepress/theme/components/landing/3. frameworks-section/FrameworksSection.vue

```diff
@@ -244,26 +244,47 @@ const numFrameworksPerRow = computed(
   () => numBlocksPerRow.value - paddedBlocksPerSide.value * 2,
 )
 
-/**
- * The indexes of the blocks on each row that support framework cards.
- */
-const centerIndexes: ComputedRef<{ start: number; end: number }> = computed(
-  () => {
-    const startIndex = paddedBlocksPerSide.value
-    return {
-      start: startIndex,
-      end: numBlocksPerRow.value - paddedBlocksPerSide.value,
-    }
-  },
-)
-
 /**
  * How many rows do we need to display all the frameworks?
  */
 const numRows: ComputedRef<number> = computed(() => {
   return Math.ceil(frameworks.length / numFrameworksPerRow.value)
 })
 
+/**
+ * The indexes of the blocks on each row that support framework cards.
```

Comment:
```suggestion
 * The indexes of the blocks on each row that support framework cards.
 *
 * Note that the index of the returned array is 1-based.
```
It took a bit for me to notice this.